ansible/roles/riakcs/tasks/admin_user.yml

```diff
@@ -12,7 +12,7 @@
 
 # Note this can't be notify because we need it to restart immediately
 - name: Restart RiakCS
-  command: riak-cs restart
+  service: name=riak-cs state=started enabled=yes
```

Comment:
Also this is making sure the service is started. It does not restart the service if it's already running ([docs](http://docs.ansible.com/ansible/service_module.html#options)). Use `state=restarted` if it should always restart the service.